scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -47,15 +48,15 @@ trait TypedSeperatedFile extends Serializable {
 /**
  * Typed tab separated values file
  */
-object TypedTsv extends TypedSeperatedFile {
```

Comment:
I agree with Ian here. We try to only use macros when we really have to. We should be able to solve this problem without a macro, since we already have a macro for the more general case (which we need to get the `Class[T]` for each column of the flattened type).